frontend/src/app/main/data/workspace/libraries_helpers.cljs

```diff
@@ -1345,6 +1367,9 @@
 
                 attr-group (get ctk/sync-attrs attr)]
 
+            (and (not= (get origin-shape attr) (get dest-shape attr))
+                 (or (not (touched attr-group)) (not omit-touched?)))
+
```

Comment:
What's the point of this block? It seems that this is an orphan condition, whose result value is not used.

Reply:
Ups! yes, is something I forgot to remove.